tests/SelfKnowledge.tex

```diff
@@ -0,0 +1,58 @@
+\section{Pepper Self Knowledge and Human Understandable Communication}
+
+This is a SSPL-only task.
+
+Pepper has to understand the meaning of actions it is being asked to perform, and describe it in a human-understandable fashion.
+
+\subsection{Focus}
+
+This test focuses on logging, high-level understanding, and human-understandable communication.
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Misc}: If any setup is required for specific tasks, the tournament officials may set such up.  Nothing is specifically required.
+\end{itemize}
+
+\subsection{Pepper Executes Commands}
+
+\begin{enumerate}
+
+	\item \textbf{Setup:} The robot is given a "Start" command.  (Teams may specify the appropriate command to the tournament official who will say it.)  At this point, the robot will start tracking what it is doing, and take note of its own actions.
+
+	\item \textbf{Commands:} Tournament officials will have prepared a series of low-level commands that are not known by teams beforehand.  These command could include standard verbal commands, or anything in the standard naoqi API, that will be fed to Pepper as a script.
```

Comment:
May I have some issues in what the challenge in letting the robot execute a sequence of predefined API calls, and then translate them to human understandable language?
Isn't it just of being aware which method has been called and then add a string += "and then I <action>"...